packages/core/src/components/Flex/Flex.tsx

```diff
@@ -70,7 +72,19 @@ const Flex: VibeComponent<FlexProps> & {
       return { gap: `var(--spacing-${gap})` };
     }, [gap]);
 
-    const overrideStyle = useMemo(() => ({ ...style, ...gapStyle }), [style, gapStyle]);
+    const flexStyle = useMemo(() => {
+      if (!flex) {
+        return {};
+      }
+
+      return {
+        flexGrow: flex.grow,
+        flexShrink: flex.shrink,
+        flexBasis: flex.basis
+      };
+    }, [flex]);
```

Comment:
If we are to enable also supplying string/number (using `CSSProperties["flex"]`):
```suggestion
    const flexStyle = useMemo(() => {
      if (!flex) return {};

      if (["string, number"].includes(typeof flex)) {
        // which means we got a valid regular css flex value:
        return { flex };
      }

      // otherwise your regular logic
      return {
        flexGrow: flex.grow,
        flexShrink: flex.shrink,
        flexBasis: flex.basis
      };
    }, [flex]);
```

Reply:
cc @tducasse @talkor

Reply:
Giving the ability to use the string is really cool IMO!

Reply:
@YossiSaadi thanks! ability to use flex as string is cool!